subsys/net/Kconfig

```diff
@@ -20,7 +20,8 @@ if NET_BUF
 config NET_BUF_USER_DATA_SIZE
 	int "Size of user_data available in every network buffer"
 	default 4
-	range 0 65535
+	range 4 65535 if BT
+	range 0 65535 if ! BT # If-statement can be removed when #7054 is resolved
```

Comment:
This doesn't scale if we want to extend this to include other exceptional defaults for other subsystems besides Bluetooth. I think the following style works (and looks) better:

```
range 0 65535
range 4 65535 if BT
```

I.e. the default range has no condition at all. This can then be extended with other `if FOO` ranges when needed.

Reply:
I was trying to be portable for the future precedence change ( #7054 ) , but I realize now that this was a mistake. Thank you for the feedback.

This is now fixed.